cli/cdsctl/config.go

```diff
@@ -119,159 +119,255 @@ func withAllCommandModifiers() []cli.CommandModifier {
 func withAutoConf() cli.CommandModifier {
 	return cli.CommandWithPreRun(
 		func(c *cli.Command, args *[]string) error {
+			// if args length equals or over context args length means that all
+			// context args were given so ignore discover conf
 			if len(*args) >= len(c.Ctx)+len(c.Args) {
 				return nil
 			}
 
-			if _, err := repo.New("."); err != nil {
```

Comment:
I extended the role of context args, in case where current directory is not a repo I also want an help to fill context args, so depending the args required it will list all projects, workflows or applications but answers will not be stored.